projects/rpx-xui-translation/src/lib/rpx-translation.service.ts

```diff
@@ -116,24 +95,9 @@ export class RpxTranslationService {
     return this.observables[phrase];
   }
 
-  public replacePhrase(phrase: string): string {
-    return phrase.replace(`_${YesOrNoValue.YES}`, '').replace(`_${YesOrNoValue.NO}`, '');
-  }
-
-  public replacePlaceholders(input: string, replacements: Replacements): string {
-    Object.keys(replacements).forEach(key => {
-      // Ideally use replaceAll here, but that isn't fully compatible with targeted browsers and packaging yet
-      const search = `%${key}%`;
-      while (input.indexOf(search) !== -1) {
-        input = input.replace(search, replacements[key]);
-      }
-    });
-    return input;
-  }
-
-  private load(phrase: string, lang: RpxLanguage, yesOrNo: string | undefined): void {
+  private load(phrase: string, lang: RpxLanguage): void {
     if (lang === 'en') {
-      this.phrases[phrase].next(phrase);
+      this.phrases[phrase].next({phrase});
```

Comment:
This needs to be a full object including yes/no (or those will be undefined when trying to display those)